packages/component/src/Utils/AbsoluteTime.js

```diff
@@ -3,17 +3,19 @@ import React from 'react';
 
 import ScreenReaderText from '../ScreenReaderText';
 
-import useLocalize from '../hooks/useLocalize';
-import useLocalizeDate from '../hooks/useLocalizeDate';
+import useDateFormatter from '../hooks/useDateFormatter';
```

Comment:
Since this is a component, should it be moved out of Utils? Doesn't have to be this PR.